mollie/api/objects/base.py

```diff
@@ -0,0 +1,7 @@
+class Base(dict):
+
```

Comment:
I'm not completely happy with the way we implemented re-using the resource thing within objects. It is profixed with an underscore especially because it's an internal structure that's not intended to be used by client consumers (only by client developers).It's also only added to classes where it's needed, so adding it to Base seemed not the right way to go.

We'll need to think about this, maybe we can find a better way to do this. But I don't think we should make it easy to use for client consumers, especially when we'll be refactoring this later on.

Reply:
Looking at it again, I don't think it's necessary to make the variable public (I thought so at first). I changed my example to reflect this.

I suppose this still isn't the best solution, but it does eliminate the visibility problem.

Reply:
Putting this on our todo list. :+1:

Reply:
Refcatored in separate PR: https://github.com/mollie/mollie-api-python/pull/51